pkg/build/builder/daemonless_test.go

```diff
@@ -26,7 +26,7 @@ func Test_mergeNodeCredentials(t *testing.T) {
 		{
 			name:    "invalid namespace credentials file",
 			nsCreds: "testdata/empty.txt",
-			errstr:  "unexpected end of JSON input",
+			errstr:  "error occurred while trying to unmarshal json",
```

Comment:
I see why we don't have this in 4.7 - my fix for Bug 1883803 got rid of the returned error.

See #180